packages/terra-clinical-data-grid/src/subcomponents/Scrollbar.module.scss

```diff
@@ -26,7 +26,7 @@
     position: absolute;
 
     .scrollbar-handle {
-      background-color: var(--terra-clinical-data-grid-scrollbar-color, #9b9fa1);
+      background-color: var(--terra-clinical-data-grid-scrollbar-handle-background-color, #9b9fa1);
       border-radius: var(--terra-clinical-data-grid-scrollbar-border-radius, 25px);
```

Comment:
Should this be --terra-clinical-data-grid-scrollbar-**handle**-border-radius,

Reply:
Okay should be switched now.